apps/web/src/server/views/applications/case/applications-case-menu.component.njk

```diff
@@ -0,0 +1,34 @@
+{% macro applicationsCaseMenu(params) %}
+	{% set application = params.application %}
+	{% set domainType = params.domainType %}
+	{% set selectedPageType = params.selectedPageType %}
+
+	<aside class="govuk-!-margin-bottom-8 pins-list-menu">
+	<ul class="gov-list pins-list-menu">
```

Comment:
better to wrap the `<UL>` on a `<NAV>` tag and add a visually hidden H3 to give context